rpm-x64/Dockerfile

```diff
@@ -37,6 +41,13 @@ RUN yum -y install \
   bzip2-devel e2fsprogs-devel file-devel libacl-devel libattr-devel \
   libxml2-devel lzo-devel nss nss-devel popt-devel sharutils xz-devel
 
+# We install our own ruby, let's remove the system one. It made rvm fail to build ruby for some reason
+RUN yum remove -y ruby
```

Comment:
We don't *need* to do this (but maybe it's a good idea to keep it anyway)

Reply:
I decided to keep it, just in case. We never ever should want to use the ruby 1.8 from the system.